pinc/gradual.inc

```diff
@@ -227,3 +228,144 @@ function maybe_output_new_proofer_project_message($project)
         echo "</div>";
     }
 }
+
+/*
+ * Output a callout and encourage the user to work in the round with the
+ * highest backlog that they can work in.
+ */
+function encourage_highest_round($username, $round_id = null)
+{
+    global $code_url, $Round_for_round_id_, $ELR_round, $ACCESS_CRITERIA;
+
+    // This URL fragment gets used a lot in this file
+    $round_url = "$code_url/tools/proofers/round.php?round_id=";
+
+    // get the backlog stats for all rounds sorted largest first
+    $backlog_stats = get_round_backlog_stats(array_keys($Round_for_round_id_));
+    arsort($backlog_stats);
+
+    // get all the rounds a user can work
+    $rounds_can_work_in = array_keys(get_stages_user_can_work_in($username));
+
+    // loop through the needs, highest to lowest, and point them to the
+    // one where they can make the most impact with their current access
+    foreach (array_keys($backlog_stats) as $round_target) {
+        $round = get_Round_for_round_id($round_target);
+
+        // don't encourage people towards $ELR_round, as that isn't really
+        // helpful for most sites
+        if ($round->id == $ELR_round->id) {
+            continue;
+        }
+
+        // if $round_id is the current one, ignore $round_id because that means
+        // we're on that round's page already
+        if ($round_target == $round_id) {
+            return;
+        }
+
+        // prepare the common wordings between the callouts
+        $please_move_up = sprintf(
+            _("Please consider proofreading a project in $round_target. Having eligible users working in <a href='%s'>%s</a> helps significantly with our backlog."),
+            "$round_url$round_target",
+            $round_target
```

Comment:
I know we control all the strings here, but may make sense to html encode in case any funcky localization issues pop upl

Reply:
That's hard because if we `html_encode()` it, the links HTML will get escaped and not render as links.